src/content/glossary.md

```diff
@@ -76,6 +76,11 @@ This index lists common terms used throughout the webpack ecosystem.
 - [__Request__](/guides/dependency-management/): Refers to the expression in the require/import statement, e.g. _require("./template/" + name + ".ejs")_, the request is _"./template/" + name + ".ejs"_.
 
 
+## S
+
+- [__Scaffolding__](/guides/scaffolding/): This feature was designed to support people who wanted to create their own configuration or initialize projects based on what other people have created.
```

Comment:
It maybe me but why are we referring to people in the past, maybe make this less time related? e.g.
```suggestion
- [__Scaffolding__](/guides/scaffolding/): This feature allows to create webpack configuration by using customizable third party initialization packages.
```